webapp/src/main/webapp/vue-apps/sidebar/components/drawer/SidebarSecondLevel.vue

```diff
@@ -24,13 +24,13 @@
     ref="secondLevelDrawer"
     v-model="drawer"
     :width="drawerWidth"
-    :style="drawerOffsetStyle"
+    :style="`background: inherit; ${drawerOffsetStyle}`"
     :right="$vuetify.rtl"
     class="HamburgerMenuSecondLevelParent border-box-sizing z-index-drawer"
     max-width="100%"
     hide-overlay>
     <v-hover v-if="drawer" v-model="$root.hoverSecondLevel">
-      <div class="full-width fill-height overflow-x-hidden overflow-y-auto specific-scrollbar">
+      <div class="full-width fill-height overflow-x-hidden overflow-y-auto specific-scrollbar transparent">
```

Comment:
Isn't needed, isn't it ?

```suggestion
      <div class="full-width fill-height overflow-x-hidden overflow-y-auto specific-scrollbar">
```

Reply:
done!